### DISCUSSION TITLE
How to write output to file using docker image

### DISCUSSION THREAD

**Discussion by HeangSok** [projectdiscovery/nuclei#1955] [Q&A] [answered]
Hello sir. I have tried this command to out put the file from your docker image, but it doesn't work:
docker run --rm -it --name nuclei_container projectdiscovery/nuclei -no-timestamp -output nucleiScanResult.txt -u {URL}

it scans well, just that it does not output the file. May you tell me how to output it?

**Comment by ehsandeep** [comment, marked as answer]
@HeangSok you can write the output to local file either using `tee` or redirecting the output to file with `>`, for example:

```
docker run projectdiscovery/nuclei -id tech-detect -u https://www.hackerone.com | tee output1.txt
```

or 

```
docker run projectdiscovery/nuclei -id tech-detect -u https://www.hackerone.com > output2.txt
```

**Reply by HeangSok** [reply]
@ehsandeep Sir, can we use the -output or -sarif-export flags instead of redirecting it?

**Reply by ehsandeep** [reply]
All those flags will write data to in docker instance and not on your local machine.

**Reply by HeangSok** [reply]
I understand now; thank you so much, sir.
